documentation/long-stay-baseline.mdx

```diff
@@ -0,0 +1,528 @@
+
+These broadly align with the correlated features explored earlier on - namely, age, arrival mode, serious illness but also include the number of previous visits, which can be considered a proxy for serious illness itself.
+
+Because the final model, using CatBoost, does not include one-hot encoding of the categorical data as CatBoost deals with this internally, we don't have further granularity on admission mode and arrival mode to compare.
+
+### 6.2 Demographic analysis<a name='demographicanalysis'></a>
+
+While the model is not peformant enough to deploy into production, it is still important to understand whether or not the model incorporates bias into its predictions.
+
+There are many kinds of bias in machine learning projects, and here we are looking at representation bias:
+
+> Does the model perform better or worse for specific categories of people across sex, ethnicity and other demographics?
+
+The specific categories are:
+
+```
+"ETHNIC_CATEGORY_CODE_DESCRIPTION", "IMD county decile", "OAC Group Name", "OAC Subgroup Name", "OAC Supergroup Name", "PATIENT_GENDER_CURRENT_DESCRIPTION", "POST_CODE_AT_ADMISSION_DATE_DISTRICT", "Rural urban classification"
+```
+
+Before looking at model performance, we need to understand how represented each category is, before drawing conclusions on categories with small sample size (note that for brevity we will only share results from `"ETHNIC_CATEGORY_CODE_DESCRIPTION", "IMD county decile","PATIENT_GENDER_CURRENT_DESCRIPTION"`):
+
+![Underlying counts for ethnicity - all data](images/long-stay-baseline/los-dist-ethnicity.png)
+
+We can see that for `ETHNIC_CATEGORY_CODE_DESCRIPTION`, the overwhelming majority of patients report `British`. We should be careful what conclusions we draw in further analysis about smaller categories, as the sample size will be very small and likely not statistically representative.
+
+![Underlying counts for sex - all data](images/long-stay-baseline/los-dist-sex.png)
+
+Sex is broadly equal, with slightly more female than male patients in this dataset.
+
+![Underlying counts for index of multiple deprivation - all data](images/long-stay-baseline/los-dist-imd.png)
+
+Index of Multiple Deprivation (IMD) deciles are skewed to the lower end, ie. there are more deprived patients present in this dataset than not.
+
+Now we can look at the distribution of length of stay for the above categories:
+
+![Underlying length of stay by ethnicity - all data](images/long-stay-baseline/los-mean-los-ethnicity.png)
+
+There is significant variation of length of stay for different ethnic groups, for example with White and black Carribean patients having a length of stay of 2.6 days on average, versus 6.0 days for Irish patients. However, as discussed previously, the count of these groups is 560 and 892 individuals respectively so further tests need to be conducted to understand if the distributions are truly different (e.g. a two-sided Kolmogorov test).
```

Comment:
...so further ^statistical hypothesis^ tests need to be conducted...
(e.g. a two-sided ^Kolmogorov–Smirnov^ test)